packages/react-board/test/setup-stage.spec.tsx

```diff
@@ -30,60 +34,80 @@ describe('setupBoardStage', () => {
 
     it('renders the canvas into a parent element', () => {
-        const { canvas, container } = setupBoard();
+        const { canvas, container } = setupBoard({
+            environmentProps: {},
+        });
 
         expect(canvas.parentElement, 'canvas was not rendered into a provided container').to.eql(container);
     });
 
     it('sets canvas height and canvas width to the provided values if no margin is provided', () => {
-        const { updateCanvas, canvas } = setupBoard();
+        const { updateCanvas } = setupBoard();
 
-        const canvasWidth = 420;
-        const canvasHeight = 690;
+        const width = 420;
+        const height = 690;
 
-        updateCanvas({
-            canvasWidth,
-            canvasHeight,
+        const canvas = updateCanvas({
+            width,
+            height,
         });
 
-        expect(canvas.offsetWidth, 'canvas width was not updated').equal(canvasWidth);
-        expect(canvas.offsetHeight, 'canvas height was not updated').equal(canvasHeight);
+        expect(canvas.offsetWidth, 'canvas width was not updated').equal(width);
+        expect(canvas.offsetHeight, 'canvas height was not updated').equal(height);
     });
 
     it('sets canvas height to auto if a "top" and "bottom" margin is provided', () => {
-        const { updateCanvas, canvas } = setupBoard();
+        const { updateCanvas } = setupBoard();
 
-        updateCanvas({ canvasHeight: 5, canvasMargin: { top: 5, bottom: 5 } });
+        const canvas = updateCanvas({ height: 5, margin: { top: 5, bottom: 5 }, backgroundColor: 'red' });
 
         expect(canvas?.offsetHeight, 'canvas height is not stretched when margins are applied').equal(
             CONTAINER_HEIGHT - 2 * 5,
         );
     });
 
     it('sets canvas width to auto if "left" and "right" margin is provided', () => {
-        const { updateCanvas, canvas } = setupBoard();
+        const { updateCanvas } = setupBoard();
 
-        updateCanvas({ canvasWidth: 5, canvasMargin: { left: 5, right: 5 } });
+        const canvas = updateCanvas({ width: 5, margin: { left: 5, right: 5 } });
 
         expect(canvas?.offsetWidth, 'canvas width is not stretched when margins are applied').equal(
             window.innerWidth - 2 * 5,
         );
     });
 
     it('sets canvas background color', () => {
-        const { updateCanvas, canvas } = setupBoard();
+        const { updateCanvas } = setupBoard();
 
-        updateCanvas({ canvasBackgroundColor: '#fff' });
+        const canvas = updateCanvas({
+            backgroundColor: '#fff',
+        });
 
         expect(window.getComputedStyle(canvas).backgroundColor, 'canvas background color was not updated').equal(
             'rgb(255, 255, 255)',
         );
     });
+    it('toggles canvas off and then on', () => {
+        const { updateCanvas, canvas: startCanvas } = setupBoard({
+            canvas: {
+                backgroundColor: 'red',
+            },
+        });
+
+        updateCanvas(undefined);
```

Comment:
just to make sure it mounts correctly

```suggestion
        expect(startCanvas.isConnected, 'canvas was not created').to.equal(true);
        
        updateCanvas(undefined);
```